examples/auth0/pages/profile.js

```diff
@@ -1,32 +1,15 @@
-import React from 'react'
-
-// This import is only needed when checking authentication status directly from getInitialProps
-// import auth0 from '../lib/auth0'
-import { useFetchUser } from '../lib/user'
 import Layout from '../components/layout'
-
-function ProfileCard({ user }) {
-  return (
-    <>
-      <h1>Profile</h1>
-
-      <div>
-        <h3>Profile (server rendered)</h3>
-        <img src={user.picture} alt="user picture" />
-        <p>nickname: {user.nickname}</p>
-        <p>name: {user.name}</p>
-      </div>
-    </>
-  )
-}
+import ProfileCard from '../components/profile-card'
+import { useFetchUser } from '../lib/user'
 
 function Profile() {
   const { user, loading } = useFetchUser({ required: true })
 
   return (
-    <Layout user={user} loading={loading}>
-      {loading ? <>Loading...</> : <ProfileCard user={user} />}
```

Comment:
Can we not remove the loading state :pray:

Reply:
It's not removed, it's handled inside the Layout component. I don't see why it has to be in all the components.